api/LinkyLink.sln

```diff
@@ -1,31 +1,31 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29613.14
-MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "LinkyLink", "src\LinkyLink\LinkyLink.csproj", "{02C51CE7-D173-4183-A75D-1C55DC1AA49C}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "LinkyLink.Tests", "tests\LinkyLink.Tests\LinkyLink.Tests.csproj", "{15A8A501-1B2F-43CB-989E-4DECEAB760B3}"
-EndProject
-Global
-	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|Any CPU = Debug|Any CPU
-		Release|Any CPU = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{02C51CE7-D173-4183-A75D-1C55DC1AA49C}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{02C51CE7-D173-4183-A75D-1C55DC1AA49C}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{02C51CE7-D173-4183-A75D-1C55DC1AA49C}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{02C51CE7-D173-4183-A75D-1C55DC1AA49C}.Release|Any CPU.Build.0 = Release|Any CPU
-		{15A8A501-1B2F-43CB-989E-4DECEAB760B3}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{15A8A501-1B2F-43CB-989E-4DECEAB760B3}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{15A8A501-1B2F-43CB-989E-4DECEAB760B3}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{15A8A501-1B2F-43CB-989E-4DECEAB760B3}.Release|Any CPU.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {EE88538F-0FA3-425F-BFC0-06E42E66B741}
-	EndGlobalSection
-EndGlobal
+﻿

```

Comment:
should this file get check in? isn't it a build artifact?

Reply:
This is the visual studio solution file... these are only whitespace changes.